moped-editor/src/components/GridTable/helpers.js

```diff
@@ -51,18 +50,28 @@ export const isFilterNullType = (operator) => {
 /**
  * Returns true if Field has a lookup table associated with it and operator is case sensitive
  * @param {string} lookupTable -  the lookup table name
- * @param {string} operator - The operator name
+ * @param {string} operator - The chosen operator name
  * @param {boolean} loading - Whether the lookup table is still loading
+ * @param {Array} lookupOperators - operators set in filter config to show autocomplete
  * @returns {boolean}
  */
 export const shouldRenderAutocompleteInput = (
   lookupTable,
   operator,
-  loading
+  loading,
+  lookupOperators
 ) => {
-  return lookupTable && !loading && AUTOCOMPLETE_OPERATORS.includes(operator);
+  return lookupTable && !loading && lookupOperators.includes(operator);
 };
 
+/**
+ * Returns true if free solo should be used for the autocomplete input with the given operator
+ * Contains operators use free solo to support both arbitrary values and values from the dropdown suggestions
```

Comment:
I think this comment is confusing so I'm thinking about how to word this better. 😅

Reply:
Operators that search on "contains" use free solo to support searching both arbitrary values and values from the dropdown suggestions 

is this any better? I was confused when I first read it b/c I thought it was the verb contains

Reply:
forgot to respond here - that is way better and i wish that i didn't remove the free solo so i could have kept it 😅